code/modules/clothing/shoes/colour.dm

```diff
@@ -57,6 +57,52 @@
 	name = "orange shoes"
 	icon_state = "orange"
 	var/obj/item/weapon/handcuffs/chained = null
+	
+/obj/item/clothing/shoes/flats
+	desc = "A pair of black, low-heeled women's flats."
+	name = "black dress flats"
+	icon = 'icons/obj/clothing/cheongsams.dmi'
+	icon_state = "dressflatsblack"
+	item_state = "dressflatsblack"
+	species_restricted = null
+	contained_sprite = 1
+
+/obj/item/clothing/shoes/flats/dressflatsred

```

Comment:
You can use an explicit statement like this: `flats/red`

There's a very low chance of any other type path matching that snippet.